### PR TITLE
Separate Modules and Plugins with Known Issues

### LINKED ISSUE #5436: Break up Modules and Plugins with Known Issues
Re: [Modules and Plugins with Known Issues](https://pantheon.io/docs/modules-plugins-known-issues)

Priority: Med

## Issue Description
This page has grown too long, with so much information that it's difficult to parse.

## Suggested Resolution
Split up the page into multiple new pages and sections:

 - Drupal Modules with Known Issues
    - Drupal 8
    - Drupal 7
- WordPress Plugins and Themes with Known Issues
    - Common Issues (Like `FS_METHOD` and assumed write access to the code base)
    - Plugins
    - Themes

Additional sections like PHP LIbraries and Dynamic Outbound IPs can be rolled into Platform Considerations or included on these pages with partial files.

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #5436 

## Effect
PR includes the following changes:
- Splits up one insanely long page into one short page and two only-sort-of-ridiculously-long pages.
- Adds a reviewdate component and review dates for every module and plugin entry.
- Consolidates plugins with assumed write access to the code base into a single table.

## Remaining Work
- [ ] CSS improvements for the review component? 
- [ ] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] ~Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)~ I don't see how that would work
- [ ] Include/exclude pages ^ respectively within docs search service provider **DO THIS**
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: gatsby/src/components/reviewDate/style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.review-date {
```
Comment (alexfornuto) [inline gatsby/src/components/reviewDate/style.css]: @carolyn-shannon Styling for the ReviewDate component is here.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5445 | Difference |
| --- | --- | --- | --- |
| Performance Score | `79` | `73` | `-6` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5445` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17795/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17795/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5445 | Difference |
| --- | --- | --- | --- |
| Performance Score | `81` | `71` | `-10` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5445` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17799/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17799/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5445 | Difference |
| --- | --- | --- | --- |
| Performance Score | `71` | `63` | `-8` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5445` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17981/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17981/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5445 | Difference |
| --- | --- | --- | --- |
| Performance Score | `80` | `68` | `-12` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5445` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17982/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17982/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5445 | Difference |
| --- | --- | --- | --- |
| Performance Score | `82` | `62` | `-20` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5445` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17993/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17993/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5445 | Difference |
| --- | --- | --- | --- |
| Performance Score | `82` | `66` | `-16` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5445` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17997/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17997/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: gatsby/src/components/reviewDate/style.css (hunk 1 above)
Comment (carolyn-shannon) [reply gatsby/src/components/reviewDate/style.css]: updated this to make more consistent - thx

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5445 | Difference |
| --- | --- | --- | --- |
| Performance Score | `81` | `67` | `-14` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `93` | `-3` |

Lighthouse Audit Reports:

- [`pr-5445` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18003/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18003/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**